ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-master.yaml

```diff
@@ -70,10 +70,9 @@ tests:
     cluster_profile: gcp
     workflow: openshift-e2e-gcp
 - as: e2e-gcp-ovn-upgrade
-  cron: '@yearly'
```

Comment:
Does the cron have this setting by default as of now? Otherwise, how often would this run?

Reply:
AIUI the cron was a hack that @dcbw put in. It's not needed now.

Reply:
So the problem was that because of how the upgrade jobs worked before they were migrated to the step registry, we couldn't get the behavior we wanted here just by defining the job in `config/`, but we couldn't override the behavior in `jobs/` if there wasn't a job defined in `config/`. One of the DPTP hackers came up with the idea of defining the job incorrectly in `config/`, but setting it to be a very-rarely-run periodic job, and then this would cause the necessary bits to be autogenerated so that we could then manually define the correct ovn-upgrade presubmit job in `jobs/`.

This presumably got broken when they rewrote the upgrade jobs to be step-based a month or two ago.